src/llscan.h

```diff
@@ -18,6 +18,19 @@ typedef std::map<uint64_t, ReferencesVector*> ReferencesByValueMap;
 typedef std::map<std::string, ReferencesVector*> ReferencesByPropertyMap;
 typedef std::map<std::string, ReferencesVector*> ReferencesByStringMap;
 
+
+// New type defining pagination options
+// It should be feasible to use it to any commands that output
+// a list of information
+typedef struct cmd_pagination {
```

Comment:
1. I'd rather have this as a class, but we can keep this as a struct.
2. `typedef` is not required for structs in C++ (C++ structs are not the same as C structs).